.gitignore

```diff
@@ -4,6 +4,8 @@ dist
 
 # Ignore config files
 /config/*
+/ux/*
```

Comment:
Following README.md there is a step:
2. `npm run ux:init` this command will copy the files and structure in MSUXF to the folder ux for futher customizing

So, that folder is not part of mern-seed project that the reason it is ignored.